less/checkbox-radio.less

```diff
@@ -139,8 +139,8 @@
         display: block;
     }
 
-    &.checkbox-lg,
-    &.radio-lg {
+    .checkbox-lg,
+    .radio-lg {
```

Comment:
It was wrong before, `.form-group` shouldn't have these classes, they should be in their child `div`. I'll go over it again and see whether it makes sense to use `>` or put it somewhere else. As we're selecting only `.checkbox-lg` and `.radio-lg` in `.form-group`, it shouldn't leak though.